### DISCUSSION TITLE
[2.7.1] Upgrading to Vuetify 2.7 from 1.5, missing dependency

### DISCUSSION THREAD

**Discussion by Knayrus** [vuetifyjs/vuetify#18789] [Q&A]
I'm currently trying to upgrade to 2.7 from 1.5 as mentioned. I'm also on Vue 2.7. I've upgraded my `eslint` and `eslint-plugin-vue`, as well as using `vue-loader: 15.11.1`. However running the server I get the error:

* vuetify/lib/components/VApp/mixins/app-theme in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/components/Main.vue?vue&type=script&lang=js

upon inspection I found that this mixin was removed from that directory, however it seems vue-loader is still trying to fetch the same.
I'm not sure if I've upgraded versions for every package/dependency required for Vuetify 2.7, so if anyone has a clue what I could be missing, it would be much appreciated.

I apologize in advance if I've missed some crucial information, this is my first time posting here. I'll provide any details that are needed.

**Comment by KaelWD** [comment]
Delete node_modules and try again.
